RealmSwift-swift2.2/LinkingObjects.swift

```diff
@@ -94,7 +94,7 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
     /// An invalidated linking objects can be accessed, but will always be empty.
     public var invalidated: Bool { return rlmResults.invalidated }
 
-    /// The number of objects in the linking objects.
+    /// The number of objects represented by the linking objects.
```

Comment:
This sentence is incredibly weird. How about just "The number of linking objects".